src/task1/task1_1.js

```diff
@@ -0,0 +1,18 @@
+const readline = require('readline');
+const util = require('util');
+
+const rl = readline.createInterface({
+  input: process.stdin
+});
+
+const question = util.promisify(rl.question).bind(rl);
```

Comment:
@EDovgalo which node version is used? Please use the latest LTS version v14.16.0
As `rl.question(query, callback)` (the latest Node LTS version) takes one callback that returns the answer, so it needs [custom promisifying](https://nodejs.org/dist/latest-v14.x/docs/api/util.html#util_custom_promisified_functions)